keyboards/gmmk/numpad/readme.md

```diff
@@ -0,0 +1,29 @@
+# GMMK NUMPAD
+
+A 17 Key macropad made ans old by Glorious LLC, equiped with a WB32 microcontroller, with support for a rotary encoder.
+
+* Keyboard Maintainer: [GloriousThrall](https://github.com/GloriousThrall)
+* Hardware Supported: GMMK Numpad
+* Hardware Availability: [GloriousPCGaming.com](https://www.gloriousgaming.com/products/gmmk-numpad)
```

Comment:
That page says that GMMK Numpad supports both wired and wireless (Bluetooth 5.0 LE) connectivity; however, the code in this PR does not seem to have any kind of wireless support.  This loss of functionality should at least be noted in the readme.